phpstan.neon

```diff
@@ -8,7 +8,7 @@ parameters:
 	reportUnmatchedIgnoredErrors: false
 	checkGenericClassInNonGenericObjectType: false
 	parallel:
-		maximumNumberOfProcesses: 4
+		# maximumNumberOfProcesses: 4
```

Comment:
Is there something wrong with the PHPSTAN's parallelization in the new version?

Reply:
I have 16 cores, so I disabled it to get feedback faster.

Any reason to have it locked to 4?